docs/_layouts/default.html

```diff
@@ -19,6 +19,18 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <title>{{ title }}</title>
+  <script>
+  (function() {
+    var path =
+      window.location.pathname +
+      window.location.search +
+      window.location.hash;
+    var match = /^\/react\/(.*)$/.exec(path);
+    if (match) {
+      window.location.replace('https://reactjs.org/' + match[1]);
+    }
+  })();
+  </script>
```

Comment:
This redirect looks good 👍 

I'm curious why we didn't remove everything from this file _except_ the redirect code though (so it would load faster) since the content isn't being rendered anymore anyway? 😁

Reply:
Some `/react/` link has been broken

Reply:
I don't understand?